README.md

```diff
@@ -6,18 +6,21 @@
 Active Directory Authentication Library for .NET (ADAL.NET) is an easy to use authentication library. You can use ADAL.NET to acquire **security tokens to access protected Web APIs**, for instance Microsoft Graph, or another Web APIs. ADAL.NET is available on various .NET Desktop/Mobile platforms to acquire a token for the signed-in user ( Windows desktop, UWP, Windows 8.1, Xamarin iOS and Xamarin Android). It can also be used in Web applications and Web APIs (ASP.NET, .NET Core, ASP.NET Core) that call other Web APIs in the name of a user, or without a user. ADAL.NET takes advantage of Windows Server Active Directory and Windows Azure Active Directory.
 
 
- Stable (`master` branch)    | Nightly (`dev` branch)
------------------------------|-------------------------
- [![NuGet](https://img.shields.io/nuget/v/Microsoft.IdentityModel.Clients.ActiveDirectory.svg?style=flat-square&label=nuget&colorB=00b200)](https://www.nuget.org/packages/Microsoft.IdentityModel.Clients.ActiveDirectory/) | [![MyGet](https://img.shields.io/myget/aad-clients-nightly/vpre/Microsoft.IdentityModel.Clients.ActiveDirectory.svg?style=flat-square&label=myget&colorB=ff0000)](https://www.myget.org/feed/aad-clients-nightly/package/nuget/Microsoft.IdentityModel.Clients.ActiveDirectory)
+| Release | Location |
```

Comment:
Hi @abhinavbose - I believe that you merged your changes (locally) and touched a lot of files. I doubt you actually meant to edit our readme.md, packaging projects etc. In fact, 238 files apear to be changed, most of which are unrelated to your change. 

Could you resolve this please and maybe start a fresh PR ? I recommend you rebase your changes on top of dev instead of merge, this is our preferred way of commiting.

Reply:
Thanks for the feedback. I have opened a new pull request: https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1102. Will close this one out.

Reply:
Seems to be #1104